test/jobs/benchmark_regression_job_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class BenchmarkRegressionJobTest < ActiveJob::TestCase
+  test "#perform benchmark_regression" do
+    benchmark_result_type = create(:benchmark_result_type)
+    benchmark_type = create(:benchmark_type)
+    commit = create(:commit)
+    initiator = commit
+    benchmark_run = create(:commit_benchmark_run,
+      benchmark_result_type: benchmark_result_type,
+      benchmark_type: benchmark_type,
+      initiator: commit
+    )
+    category = "sometime"
+    BenchmarkRegressionJob.new.perform(benchmark_run, initiator, benchmark_type, benchmark_result_type, category)
+  end
+end
```

Comment:
It isn't similar to `remote_server_job` tests. Note how those tests sets expectation about certain things.